src/recipe/recipe.service.ts

```diff
@@ -134,11 +134,13 @@ export class RecipeService {
       throw new ForbiddenException();
     }
 
-    const urls = await Promise.all(
-      files.map((file) => this.s3Service.uploadFile(file, userId, recipeId)),
+    const keys = await Promise.all(
+      files.map((key) => this.s3Service.uploadFile(key, userId, recipeId)),
```

Comment:
hmm, wasn't the previous naming better? We have an array of files but the item is called a `key`?